### PR TITLE
add RewriteRule for educationalLevel

### PR DESCRIPTION
**Pull request by sroertgen**
Hello,

we added a RewriteRule for a repository containing a controlled vocabulary.

### CODE REVIEW

File: kim/.htaccess [hunk 1]
```diff
@@ -6,4 +6,5 @@ RewriteRule ^schulfaecher/(.*) https://skohub.io/dini-ag-kim/schulfaecher/heads/
 RewriteRule ^kmk-vocabs/(.*) https://skohub.io/dini-ag-kim/kmk-vocabs/heads/main/w3id.org/kim/kmk-vocabs/$1 [R=302,L]
 RewriteRule ^license/(.*) https://skohub.io/dini-ag-kim/license/heads/main/w3id.org/kim/license/$1 [R=302,L]
 RewriteRule ^lrmi-profile/(.*) https://dini-ag-kim.github.io/amb/$1 [R=302,L]
+RewriteRule ^educationalLevel/(.*) <https://skohub.io/dini-ag-kim/educationalLevel/heads/main/w3id.org/kim/educationalLevel/$1> [R=302,L]
```
Comment (dgarijo) [inline kim/.htaccess:9]: This is not a valid URL for redirection. You should remove  "<>"

**Comment by dgarijo** [comment]
@sroertgen,
you are trying to edit a w3id for which you are not the maintainer.  Please ask [Adrian Pohl](http://lobid.org/team/ap#!) to confirm this change.

**Comment by sroertgen** [comment]
@acka47 May I ask you to confirm this change?

Thanks!

File: kim/.htaccess (hunk 1 above)
Comment (sroertgen) [reply kim/.htaccess]: Thank you. I removed the angle brackets

**Comment by dgarijo** [comment]
@sroertgen if you plan on doing other changes, I suggest you also add yourself to the readme file :)

**Comment by acka47** [comment]
> @acka47 May I ask you to confirm this change?

+1 from me and I also added @sroertgen to the README with 172a800

**Pull request merged**